content/en/docs/components/pipelines/v2/concepts/pipeline.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Pipeline"
```

Comment:
This page overlaps with https://www.kubeflow.org/docs/components/pipelines/v2/pipelines/